### DISCUSSION TITLE
你们有没有觉得Ci的配置有点复杂？

### DISCUSSION THREAD

**Discussion by Makitoid** [ClassIsland/ClassIsland#588] [Q&A] [answered]
我举一个例子：关于上下课放学提醒，我个人觉得完全可以放到一个板块：
先选择创建一个提醒表，link至一个时间表
先设置全局设置，在设置例外情况
可创建多个提醒表

**Comment by Makitoid** [comment]
我这个请求不是主要不是因为觉得配置麻烦我懒得配，而是因为我们学校有一些老师反对任何同学动电脑，他就觉得电脑是一个学习的工具，只是用来开ppt的。就拿我们班电脑举例子吧，屏幕上的文件都要快满屏了，没人过去清理整理

**Reply by wjj-8283** [reply]
不准动只是原则，但是原则又不能当饭吃，老师不在也许可以……
还有，可以冒险开个远程桌面，这样可以远程操作？

**Reply by wjj-8283** [reply]
但也许是各个学校政策不一样吧，像我自己待的班我觉得学校系统不好（win10 1909，还有冰点还原，还有一大堆乱七八糟的软件，一点都不简洁）甚至敢直接刷机（win11 ltsc，再加上一批来自github的优质开源软件），只要不干扰教学

**Reply by wjj-8283** [reply]
再者说，还有集控，如果觉得配置集控麻烦，还可以在自己的电脑上或者上微机课时完成相关配置，然后关闭受体设备上的ci，将档案文件copy到电脑上，覆盖原有的配置文件，然后重新打开，这全过程只需30s（不算配置时间）

**Reply by Makitoid** [reply]
> 不准动只是原则，但是原则又不能当饭吃，老师不在也许可以…… 还有，可以冒险开个远程桌面，这样可以远程操作？

你知道有种东西叫远程监控吗？？？
而且Ci的配置确实很麻烦啊，确是应该改进

**Reply by wjj-8283** [reply, marked as answer]
那倒是真的……我现在建新时间表只愿意复制已有的时间表再更改，不愿意从头开始做时间表
应该在程序内部内置一个模板时间表，方便一点
远程监控软件也许可以防火墙，或者火绒的联网控制
物理监控可以选择远程桌面，因为没有人出现在电脑前，只有电脑屏幕自己在动
退一万步讲，也不会有神经病24h盯着监控看吧
